### PR TITLE
Chrysler change accel pedal message to avoid false disengage on 2018 Jeeps at low speeds while using ACC.

### PR DESCRIPTION
**Pull request by adhintz**
Verified with multiple cabana drives from users.
Analysis is here: https://www.youtube.com/watch?v=Iu1e4UvAU4c

### CODE REVIEW

File: selfdrive/car/chrysler/carstate.py [hunk 1]
```diff
@@ -112,7 +112,7 @@ def update(self, cp, cp_cam):
     self.seatbelt = (cp.vl["SEATBELT_STATUS"]['SEATBELT_DRIVER_UNLATCHED'] == 0)
 
     self.brake_pressed = cp.vl["BRAKE_2"]['BRAKE_PRESSED_2'] == 5 # human-only
-    self.pedal_gas = cp.vl["ACCEL_PEDAL_MSG"]['ACCEL_PEDAL']
```
Comment (rbiasini) [inline selfdrive/car/chrysler/carstate.py]: in the dbc that's 4 bits though. Should it be 1?

Comment (adhintz) [reply selfdrive/car/chrysler/carstate.py]: I think 4 bits should be fine, as it's compared with > 0 in interface.py:
`ret.gasPressed = self.CS.pedal_gas > 0`

7 bits for ACCEL_134 is more accurate and I've submitted a PR here: https://github.com/commaai/opendbc/pull/174
Although in practice 4 bits is just fine as the low 4 bits will be set anyways as the user presses on the pedal and its value fluctuates.

Thanks for looking at the PR!

Comment (rbiasini) [reply selfdrive/car/chrysler/carstate.py]: ah perfect. Ok, I'll merge this.

**Comment by rbiasini** [comment]
thanks, this will be in the next released. Merged internally without test files and with updated opendbc.

**Pull request closed without merging**